src/adjust/adjustfiltering.cpp

```diff
@@ -146,7 +160,6 @@ QUrlQuery AdjustFiltering::filterParameters(QUrlQuery& parameters,
       continue;
     }
 
-    newParameters.addQueryItem(parameter.first, parameter.second);
```

Comment:
Do we just want to remove them completely or replace them with dummy values?

Reply:
we do not have dummy values... Better to remove them than sending invalid values.

Reply:
Ok sounds good!